ood-portal-generator/templates/ood-portal.conf.erb

```diff
@@ -179,7 +179,11 @@ Listen <%= addr_port %>
   #
   Alias "<%= @public_uri %>" "<%= @public_root %>"
   <Directory "<%= @public_root %>">
-    Options Indexes FollowSymLinks
+    ######################
+    # Removed Index from Public RootOptions as to not allow Directory Indexing
```

Comment:
We don't tend to leave annotations like this in the rendered template, mostly because it clutters things up.